tomcat-gcp-lib/pom.xml

```diff
@@ -21,14 +21,50 @@
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-datastore</artifactId>
-      <version>1.2.0</version>
+      <version>1.2.1</version>
     </dependency>
```

Comment:
Since you're explicitly adding the dependency on grpc below, should you `<exclude>` it here from the datastore dependency?